src/main/java/controllers/AdminController.java

```diff
@@ -54,4 +56,35 @@ public AdminData getAdmin(String userIdentifier) throws UserDoesNotExistError {
 
         return admin;
     }
+
+    /**
+     * Retrieves all equipment in the club's database.
+     *
+     * @return All equipment in the club's database.
+     */
+    public List<EquipmentData> getEquipment() {
```

Comment:
change name to getAllEquipment. Could have a getEquipment method to get specific equipment if we add more functionality.